### PR TITLE
fix(deps) Update starknet-reacts next to 3.0.0-beta.3 

### PR DESCRIPTION
**Pull request by Inukares**

### CODE REVIEW

File: src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+export const MintWithStarknetReact = () => {
+  const { account } = useAccount()
+  const [mintAmount, setMintAmount] = useState("10")
+
+  const [transactionStatus, setTransactionStatus] = useAtom(lastTxStatusAtom)
+  const setLastTransactionHash = useSetAtom(lastTxHashAtom)
+
+  const buttonsDisabled = ["approve", "pending"].includes(transactionStatus)
+
+  const { contract } = useContract({
```
Comment (Inukares) [inline src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx:24]: using new api from newest version of starknet-react

File: src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <Input
+          disabled
+          placeholder="Amount"
+          type="text"
+          id="mint-amount"
+          name="fname"
+          value={mintAmount}
+          onChange={(e) => setMintAmount(e.target.value)}
+        />
+        {/* TODO: When will we allow below? Need to ask Ale */}
```
Comment (Cussone) [inline src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx:76]: Not sure why would we disable it at all, after all it's just an example dapp?

File: src/app/withStarknetReactNext/_components/TransferWithStarknetReact.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  useSendTransaction,
+} from "starknet-react-core-next"
+import { useAtom, useSetAtom } from "jotai"
+import { useState } from "react"
+import Erc20Abi from "@/abi/ERC20TransferAbi.json"
+
+export const TransferWithStarknetReact = () => {
+  const { account } = useAccount()
+  const [transferTo, setTransferTo] = useState("")
+  const [transferAmount, setTransferAmount] = useState("1")
```
Comment (Cussone) [inline src/app/withStarknetReactNext/_components/TransferWithStarknetReact.tsx:18]: value `1`?

File: package.json [hunk 4]
```diff
@@ -20,16 +21,16 @@
...
     "lodash-es": "^4.17.21",
     "next": "14.2.4",
     "popmotion": "^11.0.5",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "starknet": "^6.11.0",
-    "starknet-react-chains-next": "npm:@starknet-react/chains@3.0.0-beta.2",
-    "starknet-react-core-next": "npm:@starknet-react/core@3.0.0-beta.2",
+    "starknet-react-chains-next": "npm:@starknet-react/chains@3.0.0-beta.3",
+    "starknet-react-core-next": "npm:@starknet-react/core@3.0.0-beta.3",
```
[the remark this answers is not in the document]
Comment (Inukares) [reply package.json:33]: idk to be honest, but it is a good question

File: src/app/withStarknetReactNext/_components/TransferWithStarknetReact.tsx (hunk 3 above)
Comment (Inukares) [reply src/app/withStarknetReactNext/_components/TransferWithStarknetReact.tsx:18]: Same as above

File: src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx (hunk 2 above)
Comment (Inukares) [reply src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx]: I asked Ale and blockchian team. He basically said it's always been there. Going yolo and enabling the submit 😂

File: src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+        <Input
+          type="submit"
+          disabled={true || buttonsDisabled}
+          value="Not possible with ETH!"
```
[the remark this answers is not in the document]
Comment (Inukares) [reply src/app/withStarknetReactNext/_components/MintWithStarknetReact.tsx]: I asked Ale and blockchian team. He basically said it's always been there. Going yolo and enabling the submit 😂

**Review by Cussone — APPROVED** [review]

**Pull request merged**